Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -10565,11 +10569,21 @@
             <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
             <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
             <LayoutButtons Face="BurrowDown" Type="AbilCmd" AbilCmd="AP_BurrowInfestedAbominationDown,0" Row="2" Column="4"/>
-            <LayoutButtons Face="AP_AberrationTowering" Type="Passive" Row="2" Column="0"/>
-            <LayoutButtons Face="AP_AberrationProtectiveCover" Type="Passive" Requirements="AP_HaveAberrationArmorAura" Row="1" Column="0"/>
-            <LayoutButtons Face="AP_AberrationIncubateBanelings" Type="Passive" Requirements="AP_HaveAberrationBanelingIncubation" Row="1" Column="2"/>
-            <LayoutButtons Face="AP_AberrationMonstrousResilience" Type="Passive" Requirements="AP_HaveAberrationMonstrousResilience" Row="1" Column="3"/>
-            <LayoutButtons Face="AP_AberrationConstructRegeneration" Type="Passive" Requirements="AP_HaveAberrationConstructRegeneration" Row="1" Column="1"/>
+            <LayoutButtons index="5" AbilCmd="AP_BurrowInfestedAbominationDown,Execute"/>
+            <LayoutButtons index="6" Face="AP_AberrationProtectiveCover" Requirements="AP_HaveAberrationArmorAura" Row="1"/>
+            <LayoutButtons index="7" Face="AP_AberrationIncubateBanelings" Requirements="AP_HaveAberrationBanelingIncubation" Column="2"/>
+            <LayoutButtons index="8" Face="AP_AberrationMonstrousResilience" Requirements="AP_HaveAberrationMonstrousResilience" Column="3"/>
+            <LayoutButtons index="9" Face="AP_AberrationConstructRegeneration" Requirements="AP_HaveAberrationConstructRegeneration" Column="1"/>
+            <LayoutButtons index="10">

```

Comment:
Looks like something went weird here with the buttons; I don't know why it's trying to include `index="` in there, and that means `AP_BurrowInfestedAbomincationDown` is essentially getting defined and then modified one line below.

The `index="5"` line should be merged with the line above (I think just change `,0` to `,Execute`?), and then `index=""` removed from all the following lines. `index="10"` would be nice to have on a single line for style reasons, but that's optional.

Reply:
You shouldn't use indices here as they're for overriding stuff. This code creates a new unit with the most default parent, therefore there's no reason to override anything